src/pod/personal-storage.ts

```diff
@@ -42,7 +41,7 @@ export class PersonalStorage {
    * @param name pod name
    */
   async create(name: string): Promise<Pod> {
-    assertActiveAccount(this.accountData)
+    assertAccount(this.accountData)
```

Comment:
typescript fails here     when `for` with  `async/await`, see            if `.bind` fixes it

Reply:
I have a test where this method is called in a loop and it works. Can you show an example that it doesn't work for you?